filebeat/docs/fields.asciidoc

```diff
@@ -12056,14 +12056,24 @@ URL or file name for a threat.
 
 --
 
-*`palo_alto.pan_os.threat_id`*::
+*`palo_alto.pan_os.threat.id`*::
 +
 --
 type: keyword
 
 Palo Alto Networks identifier for the threat.
 
 
+--
+
+*`palo_alto.pan_os.threat.name`*::
++
+--
+type: keyword
+
+Palo Alto Networks name for the threat.
```

Comment:
`s/Networks/Network’s/g`?

Reply:
I guess if changed it should be `Palo Alto Networks' name`, or `Palo Alto Networks's name`?

In their own docs they write it like `Palo Alto Networks identifier for the threat.` so I'd rather leave it as is

Reply:
If that’s how they like to write it then leave it as is.